models/src/main/scala/it/pagopa/interop/agreementmanagement/model/agreement/PersistentAgreement.scala

```diff
@@ -18,5 +18,6 @@ final case class PersistentAgreement(
   suspendedByPlatform: Option[Boolean],
   consumerDocuments: Seq[PersistentAgreementDocument],
   createdAt: OffsetDateTime,
-  updatedAt: Option[OffsetDateTime]
+  updatedAt: Option[OffsetDateTime],
+  details: Option[String]
```

Comment:
the name is too generic, may we use something like `consumerNotes`? 
Or anything better that you may find

Reply:
I was waiting for your comment to decide on the field name :trollface: